src/assets/icons/food/icon-taco.vue

```diff
@@ -1,12 +1,16 @@
 <template>
-  <svg version="1.1" xmlns="http://www.w3.org/2000/svg"
-       xmlns:xlink="http://www.w3.org/1999/xlink"
-       x="0px"
-       y="0px"
-       viewBox="0 0 512.001 512.001" style="enable-background:new 0 0 512.001 512.001;"
-       xml:space="preserve">
-
-		<path d="M458.03,215.786c-2.949-2.907-7.698-2.873-10.605,0.076c-2.908,2.95-2.874,7.699,0.076,10.606
+  <svg
+    version="1.1"
+    xmlns="http://www.w3.org/2000/svg"
+    xmlns:xlink="http://www.w3.org/1999/xlink"
+    x="0px"
+    y="0px"
+    viewBox="0 0 512.001 512.001"
+    style="enable-background:new 0 0 512.001 512.001;"
```

Comment:
move inline styles to `styles` section if possible, inline styles may cause some problems with specificity

Reply:
this was added to the icon by default. when i was messing with it in other components they were causing issues